### ISSUE TITLE
New Tab Page settings

### ISSUE THREAD

**Issue by rebron** [brave/brave-browser#2999]
<img width="677" alt="screen shot 2019-01-16 at 9 56 30 pm" src="https://user-images.githubusercontent.com/5951041/51298510-a9dde500-19d9-11e9-86b3-7ebb653964b1.png">

## Description 
Create new setting 'New tab page shows' with drop down menu:
a) Dashboard with images
b) Homepage
c) Blank page

Related to: https://github.com/brave/brave-browser/issues/960
and should show in the Get Started section

**Comment by petemill** [comment]
We actually need the browser implementation for this as well. As in there is no functionality at the moment for new tabs to open to a specified URL. I’m not sure how much of a priority this would be for our users? Especially since an extension could fulfill that need.

**Comment by rebron** [comment]
Carryover as a muon feature parity. Separating out from the other issue due to actually having the functionality available.

**Comment by rebron** [comment]
We should probably reconsider placement. I'd rather these settings be available on the new tab page where it's accessible and relevant than buried in Settings. cc: @karenkliu, @cezaraugusto

**Comment by eljuno** [comment]
+1s from Community https://community.brave.com/t/feature-request-empty-new-tab-page/39676?u=eljuno

**Comment by cezaraugusto** [comment]
agree with @rebron. cc @karenkliu for ideas

**Comment by srirambv** [comment]
++ on @rebron's idea of having it in new tab itself

**Comment by tomlowenthal** [comment]
We could have it both places.

**Comment by franzjschneider** [comment]
+1 from me on this. Blank new-tab pages are about all I use. Let clutter, less resources, less distraction. Simple.

**Comment by ryder203** [comment]
tl;dr ... +1 need options set custom URL or page/blank/edit wallpaper background

I installed Brave just now on my Pixel and my Windows 10 Machine. Imported all my bookmarks, made a sync (cool feature), had a look at all my plugins in Chrome, installed them manually on Brave, did all the settings stuff I need on my Android and Windows system for browsing. Uninstalled Bromite on my Android and Chrome on my Windows. I really enjoyed Brave and thought I do not need anything else now. And then I realized that on Brave for Android and for Windows I do not have any options for the new tab page - and I see that (sorry) ugly page with useless information when opening a new tab. Options for the new tab page are essential. That feature exists in Chrome and other browsers for a good reason. This is a real letdown, and even at least on Windows I could add a plugin to override, it is not OK not to have those options. I wonder why a project that wants to create somehting better on the basis of chromium/chrome removes such an important feature. Sigh.

**Comment by dpedu** [comment]
+1. It's rediculous that a browser does not allow us to set a home page.

**Comment by rebron** [comment]
@dpedu I think you mean for new tab page, we're working on that. For homepage/home button in settings you can set your homepage there when you show the home button. And on startup, you can specify the page you want.

**Comment by dpedu** [comment]
@rebron what I mean by home page is the screen shown when opening a new tab, be it in a new window, tab in an existing window, or when I open the browser. Some of these can be changed w/o an extension, yes, others not.

**Comment by StormFalcon32** [comment]
@dpedu Yeah that's just called new tab. Homepage is something else entirely.

**Comment by StormFalcon32** [comment]
I wish that the newtab was identical to the chrome one or that it at least changed backgrounds with the theme. Currently, themes can't change the newtab background, which is 50% the reason I even use themes in the first place

**Comment by srirambv** [comment]
+1 from @ChildishGiant via #5114 for setting new tab to a URL

**Comment by Valinwolf** [comment]
+1
I never understood why Chrome didn't have the ability to set the new tab page without an extension. To me it's an essential feature. I shouldn't have to see that redirect page before my desired URL from an extension or create my own to override the new tab page with a specific URL. Plus it requires an additional layer of trust to be made, instead of just trusting the browser you have to trust the browser and the extension. I hope brave gets this done soon because the choice of rainbow colors, cycling pictures, or an extra extension is not appealing. I've been using brave-nightly-bin from AUR in hopes that it would be implemented soon. 😕

**Comment by gaelbouvyer** [comment]
When I open a new tab, I would love it to be automatically with a specific URL (web design inspiration in my case).

**Comment by Jos512** [comment]
+1

We definitely need this feature, also because of the speed and usability of Brave. On my computer there's a significant difference in how fast new tabs open in regular Brave and in Private Mode. And that's with the same extensions in both.

A blank new tab page is a bit boring, but very responsive to work with.

**Comment by Valinwolf** [comment]
Do we have any progress? This has been ~10 months since original post without any indication on whether it's being worked on and how much progress has been made.

**Comment by yochem** [comment]
PLEASE add this. I just want to have the dark mode color as background when opening a new tab 😢. Some images are not so nice on the eyes

**Comment by 3rd-Eden** [comment]
So is it safe to assume that this issue has absolutely no priority from the Brave team because they use it to inject their own ads into this dashboard and would miss out of revenue if this feature was implemented? Because I honestly don't believe it's that hard to read a user defined URL when a new tab is opened.

**Comment by chopin2712** [comment]
+1 Really, it's needed. More than that there are now sponsored images on the dashboard and they are even more ugly !!!

**Comment by StormFalcon32** [comment]
> +1 Really, it's needed. More than that there are now sponsored images on the dashboard and they are even more ugly !!!

You can get rid of those

**Comment by jonathansampson** [comment]
@chopin2712 If you'd like to hide the Sponsored Images, you can do so from the New Tab Page. At the bottom-right you'll find four icons, the first of which is for Dashboard Settings. Click that, and there you will see the option to disable Sponsored Images. I hope this helps!

**Comment by mrose17** [comment]
and *please* include a setting to indicate whether i want the 12-hour or 24-hour time on the NTP... cc: @aaronsantamaria ... thank you!

**Comment by bsclifton** [comment]
@rebron should we revisit this after @cezaraugusto recently added the new section? 😄 

I think it wouldn't be too hard to add this functionality:
![Screen Shot 2020-04-15 at 1 00 50 AM](https://user-images.githubusercontent.com/4733304/79313025-99563c00-7eb4-11ea-88f2-3550f57d0f9b.png)

**Comment by mrose17** [comment]
That's looks very nice. though, please, let me customize the clock for AM/PM v. 24H ... thank you!

**Comment by madhums** [comment]
Would be great to have a dark background as an option! (Is that difficult to include?)

**Comment by dpedu** [comment]
I just noticed this issue is still around, so I looked up some fun numbers. The original release of the first Netscape browser was developed over less than 22 months and supported setting a default page. This issue has been open for about 15 months.

**Comment by GlitchMC** [comment]
I'd like to know what's up with this, too. Just installed and set up, went through all my settings, installed a theme, only to find that the New Tab page doesn't actually show my theme.
Very unfortunate. Back to chrome I guess.

**Comment by madhums** [comment]
@bsclifton @rebron any update on a simple plain blank page?

**Comment by Sa-Ja-Di** [comment]
@bsclifton @rebron Would really be nice if you guys for once would  stop on minimalism and simplicity and do instead some serious work of which customization requiring power-users would contribute.

For example, Navigation toolbar customization (moving buttons), or Treestyle tabs or something like an optional status bar or things as simple like new tab/dashboard settings which can be customized.

**Comment by bsclifton** [comment]
@Sa-Ja-Di vertical / tree-style tabs captured with https://github.com/brave/brave-browser/issues/464 in case you wanted to give it a +1

@simonhong might be able to check this out soon 😄

**Comment by OtsoT** [comment]
+1 I honestly just don't want sponsored images with sponsored links on my new tab page. Especially the recent "Save the Children" charity rubs me the wrong way. Pretty sure some countries have heavy regulations of using images of children in advertisement (and it's technically also advertisement for nexo.io).

**Comment by ChildishGiant** [comment]
@OtsoT Luckily you can turn off the sponsored images
![image](https://user-images.githubusercontent.com/13716824/80115135-46e3e200-857c-11ea-913f-b93123fb9e1d.png)

**Comment by brandneetjan** [comment]
Vimium doesn't work on Braves new tab page so I'd really prefer to change it into the homepage of my search engine. Is this because of greed by brave that I can't just add any url as a new tab page? I might even switch browsers because of this tbh.

**Comment by brandneetjan** [comment]
Vimium doesn't work on Braves new tab page so I'd really prefer to change it into the homepage of my search engine. Is this because of ad revenue greed by brave that I can't just add any url as a new tab page? I might even switch browsers because of this tbh.

**Comment by chakrihacker** [comment]
Any Update on this. I kind of looking for options to have duckduckgo(any other search engine) in the middle of new tab page or an option to make search engine the default tab page

**Comment by wilfra** [comment]
Having just downloaded Brave I am shocked this feature is not available. It seems quite deliberate as a revenue generator as you have to uncheck a slew of defaults simply to get the page to not show all of the default content on it.

Bad UX...

**Comment by bsclifton** [comment]
@wilfra we're working on a fix now - patch is up here for review (thanks @simonhong):
https://github.com/brave/brave-core/pull/6879

This is likely going to be merged to 1.18 which should hit Release channel ~December 8th

**Comment by bsclifton** [comment]
Live in Nightly! 😄 Great job, @simonhong 
![image](https://user-images.githubusercontent.com/4733304/97811293-d3abde80-1c36-11eb-8bcd-56a840499435.png)

**Comment by GeetaSarvadnya** [comment]
Verification passed on

```

Brave | 1.18.69 Chromium: 87.0.4280.88 (Official Build) (64-bit)
-- | --
Revision | 89e2380a3e36c3464b5dd1302349b1382549290d-refs/branch-heads/4280@{#1761}
OS | Windows 10 OS Version 2004 (Build 19041.630)

```
Verified the test plan from https://github.com/brave/brave-core/pull/6879

**Comment by pitsi** [comment]
Noob question and sorry for bringing this back up.
I am interested in setting a "homepage" (= an html file I have locally) there. So. how do I set that homepage that will open in every new tab?

**Comment by simonhong** [comment]
@pitsi That `homepage` option is related with below homepage option.
If homepage is configured, new tab will show that url. Loading local html file in new tab is not supported.

![image](https://user-images.githubusercontent.com/6786187/101964855-14474080-3c56-11eb-88bb-46187d0b617b.png)

**Comment by pitsi** [comment]
Thank you. Well hidden setting :P 

On my end, I can set a local file there (file:///home/user/whatever.html) and it works as a regular page from the net. 
Plus, once I set that homepage under show home button, I can disable the home button again and the page still works!
Until today, I was running lighttpd on my system to serve that page locally and I had set it as a page for newtab with new tab redirect. Now I can remove them both :)

**Comment by simonhong** [comment]
@pitsi Great tip!

**Comment by raiden-e** [comment]
> Thank you. Well hidden setting :P
> 
> On my end, I can set a local file there (file:///home/user/whatever.html) and it works as a regular page from the net. Plus, once I set that homepage under show home button, I can disable the home button again and the page still works! Until today, I was running lighttpd on my system to serve that page locally and I had set it as a page for newtab with new tab redirect. Now I can remove them both :)

I tried this, but setting the homepage to `/Users/user/source/startpage/index.html` won't do it on MacOS. :/

**Comment by pitsi** [comment]
I just tried in on a... quite fresh installation (3 months old) of brave I have on my laptop and it works. 
Are you sure you do not have any typos, or that the path is correct or that you have also changed the new tab to show the homepage (it is at the bottom of brave://settings/getStarted).

Also, does it work with online urls like this?
`https://instant.unsplash.com/`

**Comment by raiden-e** [comment]
I copied the link from the search bar, so I figure it's correct. I might add, that it DOES work when starting Brave. However, opening a new tab will open the dashboard, instead of the HTML-URL. The setting "New tab page shows" is set to "Homepage". So extra odd.
How about you?

**Comment by pitsi** [comment]
As I said above, do you have set brave to show the homepage on the new tab?

**Comment by raiden-e** [comment]
Yes :)

**Comment by pitsi** [comment]
No idea then. Here is how I have set mine. The "show home button toggle" was enabled so as to show the options below it. I have it disabled otherwise.
![2024-09-27-195013_1280x1024_scrot](https://github.com/user-attachments/assets/4dd677d8-fe3d-47ad-b39b-e4d55ddb7e9f)
![2024-09-27-195135_1280x1024_scrot](https://github.com/user-attachments/assets/6f6c6d89-ce80-4b54-99df-a34feece4631)
